src/main/java/com/novi/backend_spring_boot_tech_it_easy_controller/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,38 @@
+package com.novi.backend_spring_boot_tech_it_easy_controller.controllers;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+public class TelevisionsController {
+    @GetMapping("/televisions")
+    @ResponseBody
+    public String getAllTelevisions() {
+        return "List Televisions";
+    }
+
+    @GetMapping("/televisions/{model}")
+    @ResponseBody
+    public String getTelevision(@PathVariable String model) {
+        return "Type television" + model;
```

Comment:
Spring returned automatisch een HttpResponse. Jij returned hier een String, maar dat wordt dus de body van het HttpResponse. De status wordt automatisch 200 (ok), maar het is verstandig om daar ook in dit geval controle over te nemen door ook hier een ResponseEntity te returnen.